panct/__main__.py

```diff
@@ -1,25 +1,63 @@
 #!/usr/bin/env python
+
 import typer
+from typing_extensions import Annotated
+from .complexity import AVAILALBE_METRICS
 
 app = typer.Typer()
 
 
 @app.command()
-def complexity(gfa_file: str, output_file: str):
+def complexity(
+    output_file: Annotated[str, typer.Option("--out", help="Name of output file")],
+    gfa_file: Annotated[
+        str, typer.Option("--gfa", help="Path to .gfa file of the graph")
+    ] = "",
+    gbz_file: Annotated[
+        str, typer.Option("--gbz", help="Path to .gbz file of the graph")
+    ] = "",
```

Comment:
since either one or the other of these is required, it might be simpler to combine them into a single argument? and then just check the file extension to figure out which one it is?

also, since this is a required input, maybe we should make it a positional arg instead of an optional arg? Typer recommends using optional args only for optional inputs and positional args for requires ones. I think that is [the convention](https://typer.tiangolo.com/tutorial/options/required/#required-cli-options) in most UNIX tools (albeit one that is often-ignored). In theory, it should make it easier for someone to figure out what the required inputs are from a quick glance